### ISSUE TITLE
Only getting half the output from the mixer centrifuge that I expect

### ISSUE THREAD

**Issue by gchpaco** [EpicSquid/Embers#16]
Noticed this making dawnstone; I'm running EmbersRekindled-1.2.jar and the Soot 1.0 that is calibrated to work with it, and when I feed 1 gold and 1 copper into the mixer centrifuge, I get 1 dawnstone out.  JEI says that I should get 2 dawnstone.

**Comment by gchpaco** [comment]
Lemme try reproducing with a smaller pack than my kitchen sink.

**Comment by wamc2017** [comment]
I guess that's a duplicate of #5

**Comment by gchpaco** [comment]
Yes, I believe wamc has it.  This is almost certainly the same bug as #5.

**Comment by DaedalusGame** [comment]
oh huh, i thought i fixed that before 1.2
